sdk/reya_websocket/resources/market.py

```
@@ -142,6 +160,59 @@ def for_symbol(self, symbol: str) -> "MarketSummarySubscription":
         return MarketSummarySubscription(self.socket, symbol)
 
 
+class AllSpotMarketsSummaryResource(SubscribableResource):
+    """Resource for accessing all spot markets summary data."""
+
+    def __init__(self, socket: "ReyaSocket"):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)